README.md

```diff
@@ -4,8 +4,8 @@
 [![Stackage version](https://www.stackage.org/package/Cabal/badge/lts?label=Stackage)](https://www.stackage.org/package/Cabal)
 [![Artifacts](https://github.com/haskell/cabal/actions/workflows/artifacts.yml/badge.svg?branch=master)](https://github.com/haskell/cabal/actions/workflows/artifacts.yml?query=branch%3Amaster)
 [![Documentation Status](http://readthedocs.org/projects/cabal/badge/?version=latest)](http://cabal.readthedocs.io/en/latest/?badge=latest)
-**[IRC](https://web.libera.chat/#hackage)**
-**[Matrix](https://matrix.to/#/#hackage:libera.chat)**
+[![IRC chat](https://img.shields.io/badge/chat-on%20libera-brightgreen.svg)](https://web.libera.chat/#hackage)
+[![Matrix chat](https://img.shields.io/badge/chat-on%20matrix-brightgreen.svg)](https://matrix.to/#/#hackage:libera.chat)
```

Comment:
```suggestion
[![IRC chat](https://img.shields.io/badge/chat-via%20libera-brightgreen.svg)](https://web.libera.chat/#hackage)
[![Matrix chat](https://img.shields.io/badge/chat-via%20matrix-brightgreen.svg)](https://matrix.to/#/#hackage:libera.chat)
```

Reply:
"via IRC" maybe? people might not know that libera is an irc server. sorry for the nitpicking, feel free to merge if you disagree